tools/hive-bigquery/custom_exceptions.py

```diff
@@ -0,0 +1,27 @@
+"""Defines custom exceptions"""
+
+
+class ConnectionError(Exception):
+    """Raised when there is a connection failure to any of the Hive, BigQuery,
+    GCS, and MySQL components"""
+    pass
+
+
+class MySQLExecutionError(Exception):
```

Comment:
Why not use `mysql.connector.Error`

Reply:
This custom exception will handle multiple lower level 'pymysql' errors by grouping them.